src/unitxt/inference.py

```diff
@@ -425,6 +423,24 @@ class WMLInferenceEngine(
     data_classification_policy = ["public", "proprietary"]
     parameters: Optional[WMLInferenceEngineParams] = None
 
+    _client: Any = None
+
```

Comment:
```suggestion
def verify(self):
     super().verify()
     if self.credential is not None:
         for key in self.credential:
             if key not in ["url", "apikey", "project_id"],:
                 raise ValueError(f'Illegal credential key: {key}, use only ["url", "apikey", "project_id"]')
```
Better to make sure no one made mistakes here